testdata/src/example.go

```diff
@@ -59,6 +59,12 @@ func example() {
 
 		break
 	}
+
+	// detects contexts wrapped in function literals (this is risky as function literals can be called multiple times)
+	_ = func() {
+		ctx = wrapContext(ctx) // want "nested context in function literal"
```

Comment:
` // want "nested context in function literal"` tells the `analysistest` library what lint message to expect for the line.